src/commands/delete.js

```diff
@@ -0,0 +1,48 @@
+const utils = require('../utils');
+
+const _delete = (context, version) => {
+  if (!version) {
+    context.line('Error: No version - provide a version like "1.0.0"...\n');
+    return Promise.resolve(true);
+  }
+  return utils.checkCredentials()
+    .then(() => utils.getLinkedApp())
+    .then((app) => {
+      context.line(`Preparing to delete version ${version} of your app "${app.title}".\n`);
+      const url = `/apps/${app.id}/versions/${version}`;
+      utils.printStarting(`Deleting ${version}`);
+      return utils.callAPI(url, {
+        method: 'DELETE',
+      });
+    })
+    .then(() => {
+      utils.printDone();
+      context.line('  Deletion successful!\n');
+    });
+};
+_delete.argsSpec = [
+  {name: 'version', example: '1.0.0', required: true, help: 'the version to delete'},
+];
+_delete.argOptsSpec = {};
+_delete.help = 'Delete a version of your app as long as it has no users/Zaps.';
+_delete.example = 'zapier delete 1.0.0';
+_delete.docs = `\
+A utility to allow deleting app versions that aren't used.
+
+> The app version needs to have no users/Zaps in order to be deleted.
+
+**Arguments**
+
+${utils.argsFragment(_delete.argsSpec)}
+${utils.argOptsFragment(_delete.argOptsSpec)}
+
+${'```'}bash
+$ zapier delete 1.0.0
```

Comment:
Great questions! I pondered doing `zapier versions 1.0.0 --remove` but since the process to add a version is to use `zapier push`, it seemed fitting to have a separate command to remove (unlike `zapier invite` or `zapier env` where we use that same command to add).

For 1) and 2) are you suggesting a `zapier delete-app` or `zapier delete-version 1.0.0`, for example? I'd definitely prefer `zapier delete app` and `zapier delete version 1.0.0` there.